balances/src/lib.rs

```diff
@@ -260,6 +273,13 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
+	/// Remove an account from a community
+	fn remove_account(cid: CommunityIdentifier, who: &T::AccountId) -> DispatchResult {
+		<Balance<T>>::remove(cid, who);
```

Comment:
this is a critical, destructive function. please ensure that the balance is below ED before removing the account 

even if the current usage ensures that, we should not assume that devs know what they're doing ;-)

Reply:
haha ok done

Reply:
uuh good catch, i forgot to remove the call to `remove_account` in `transfer_all`

Reply:
also added a check